### PR TITLE
Improved hlm

### PR DESCRIPTION
**Pull request by pranavbaburaj**
I created an improved version of hlm in python

## How to use it?
Install packages
`python hlm.py install <package-name>`

Uninstall packages<br>
`python hlm.py install <package-name>`

<hr>

These packages are installed to <dir>/hlib_modules directory from where they can be imported

### CODE REVIEW

File: hlm.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import click
+import os
+from clint.textui import colored as color
+import requests
```
Comment (pranavbaburaj) [inline hlm.py:4]: imports

**Comment by pranavbaburaj** [comment]
I have also created a release for hlm. 
It is lightweight. Check it out [here](https://github.com/pranavbaburaj/hlm/releases/download/1.0/hlm.exe)

**Comment by hasan-kashi** [comment]
Hlm most written with hascal ,not python

**Comment by hasan-kashi** [comment]
Hlm most written with hascal ,not python

**Comment by hasan-kashi** [comment]
You can add your hlm repo in awesome hascal repo with a pull request.
[awesome hascal](https://github.com/hascal/awesome-hascal)

**Comment by pranavbaburaj** [comment]
Ok, Thank you

**Pull request closed without merging**